packages/berry-parsers/sources/grammars/shell.pegjs

```diff
@@ -26,6 +26,11 @@ CommandChainThen
 CommandChainType
   = '|&'
   / '|'
+  / '>'
+  / '>>'
+  / '<'
+  / '<<
```

Comment:
`<<` doesn't exist in bash, does it? 🤔

Reply:
It does, although probably not as commonly used https://www.gnu.org/software/bash/manual/html_node/Redirections.html#Redirections. I think we'll also end up with couple more variations `>>>`, `&>`, etc.

Reply:
Oh right HEREDOC! Not sure we'll support them though since they're typically multiline (and the portable shell is mostly useful for one-liner in the `scripts` field, not full-fledged scripts).

That said `<<<` will be a great addition later on, I agree 👍